### ISSUE TITLE
Highland map is generated without ruins or luxuries

### ISSUE THREAD

**Issue by Der-Zorn-Gottes** [LoneGazebo/Community-Patch-DLL#9304]
_1. Mod Version (X.Y.Z). Current Version: 2.7.3_
Community Patch v 110
Vox Pomuli v 15
EU compabilty v 1
Promotion icons v 6
Promotion tree v 22

_2. DirectX Version_

_3. List of Other Mods_

None

_4. Describe the Issue_

The map highlands is generated without luxuries and ancient ruins, Did not play long enoufg to confirm on strategic resources.
Checked continent map, seemd fine.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by nobubblegums** [comment]
@azum4roll

**Comment by azum4roll** [comment]
Fixing all is feasible, but I need a map list and someone to add them to the VP modinfo file after I fix them all.

**Comment by nobubblegums** [comment]
Oh cool. I added PangaeaPlus and ContinentsPlus for the next version, and we have a Terra script (from More Luxuries iirc), I'm not sure if it overwrites the game's Terra tho. Anyway, I'd say these three (esp. PangaeaPlus) would get the most use out of that list. People have been complaining about CS placement on "Plus" maps, maybe that's an area for improvement.
Leaving them aside, if you can think of any improvements&VP optimizations to the base Pangaea map, people would love that, think it's the second most popular map after communitu.

**Comment by azum4roll** [comment]
Is it ok if I just remove the whole part of "assigning all CS outside of regions" in plus maps?

**Comment by Der-Zorn-Gottes** [comment]
Just to make it clear: I have played highlands without any issues for years. It's only been since the last patch that things are messed up.

**Comment by nobubblegums** [comment]
Yeah, no need to remove it if it's identical to standard Pangaea in other ways.

**Comment by azum4roll** [comment]
It's commented to be "better than standard maps" and the CS thing is a downside. I think these maps make better islands?

**Comment by azum4roll** [comment]
There should be a way to replace the original scripts in modinfo?

**Comment by jayeye2011** [comment]
Because I play single player using a MP_MODPACK (due to faster load time) the VP installation map folders included maps are not visible in-game unless I manually copy the VP maps script I want to use to my user documents\My Games\Sid Meier's Civilization 5\maps folder. As a result, both the vanilla map and the VP map are listed in the game map selection, so the only way to differentiate them is by the name that is used in the GetMapScriptInfo() function in the lua file.
